src/patients/util/validate-patient.ts

```diff
@@ -12,7 +12,7 @@ const validateEmails = (emails: ContactInfoPiece[] | undefined) =>
 
 const validatePhoneNumbers = (phoneNumbers: ContactInfoPiece[] | undefined) =>
   (phoneNumbers ?? []).map((phone) =>
-    !validator.isMobilePhone(phone.value) ? 'patient.errors.invalidPhoneNumber' : undefined,
+  (!validator.isMobilePhone(phone.value) || !new RegExp(`^[\\s\\d]{${10}}$`).test(phone.value)) ? 'patient.errors.invalidPhoneNumber' : undefined,
```

Comment:
Declaring a new object within a ternary operator makes it a bit hard to read...maybe we can do something like this:
```
const validPhoneRegex = /^[\\s\\d]{${10}}$/

const validatePhoneNumbers = (phoneNumbers: ContactInfoPiece[] | undefined) =>
  (phoneNumbers ?? []).map((phone) =>  
    (!validator.isMobilePhone(phone.value) || !validPhoneRegex.test(phone.value)) ? 'patient.errors.invalidPhoneNumber' : undefined,  
)
```

Another option you have is to create a static utility method to do the regex checking, but that isn't necessary here.